public/manifest-shim.json

```diff
@@ -0,0 +1,4 @@
+{
+    "bypass-url-scheme-list": ["wc"],
+    "bypass-url-domain-list": ["arweave.net"]
```

Comment:
maybe `external-uri` then 

```json
external-uri: [
    "wc:*" // Wallet Connect,
    "https?://*.arweave.net/*"
]
```

Reply:
iOS / Android not support JSON with Comment.